indico/modules/events/registration/fields/base.py

```diff
@@ -14,7 +14,12 @@
 from indico.util.marshmallow import not_empty
 
 
-class BillableFieldDataSchema(mm.Schema):
+class FieldSetupSchemaBase(mm.Schema):
```

Comment:
this will need some validation later, e.g. to check that the field id belongs to the same form (technically it would not be harmful to not have that check, but it's super messy if you end up - for whatever reason - with references to fields from other events)